src/zelos/api/memory_api.py

```diff
@@ -17,8 +17,9 @@
 
 import ctypes
 
-from typing import List
+from typing import List, Optional
 
+from zelos.emulator.base import MemoryRegion
```

Comment:
Import MemoryRegion into Zelos so we don't refer to emulator.base

Reply:
As discussed, this has to stay the same because of circular dependency. But plugins, for example, can now import `MemoryRegion` directly from `zelos`.